aldryn_config.py

```diff
@@ -200,7 +203,7 @@ def security_settings(self, data, settings, env):
         s['INSTALLED_APPS'].append('djangosecure')
```

Comment:
ah, yes. Serves me right for not actually re-building the whole docker image while testing....